include/boost/geometry/srs/projections/proj/omerc.hpp

```diff
@@ -99,7 +102,8 @@ namespace projections
                         if(fabs(temp) < tolerance) {
                             u = this->m_proj_parm.A * lp_lon;
                         } else {
-                            u = this->m_proj_parm.ArB * atan2((s * this->m_proj_parm.cosgam + V * this->m_proj_parm.singam), temp);
+                            u = this->m_proj_parm.ArB * atan2((s * this->m_proj_parm.cosgam
+                                + V * this->m_proj_parm.singam), temp);
```

Comment:
Here is another place. There are also other places below. I won't point them all out. The comment I made above about the original proj code applies here and below too.